### PR TITLE
fixed a bug; print at stage14

### PR DESCRIPTION
**Pull request by SunSki**
I made it available in python3

### CODE REVIEW

File: egs/swbd/s5c/local/chain/multi_condition/run_tdnn_aug_1a.sh [hunk 1]
```diff
@@ -128,7 +128,7 @@ if [ $stage -le 14 ]; then
   echo "$0: creating neural net configs using the xconfig parser";
 
   num_targets=$(tree-info $treedir/tree |grep num-pdfs|awk '{print $2}')
-  learning_rate_factor=$(echo "print 0.5/$xent_regularize" | python)
+  learning_rate_factor=$(echo "print(0.5/$xent_regularize)" | python3)
```
Comment (danpovey) [inline egs/swbd/s5c/local/chain/multi_condition/run_tdnn_aug_1a.sh]: this works the same in python2 and python3, I don't see why the change would be needed.

Comment (SunSki) [reply egs/swbd/s5c/local/chain/multi_condition/run_tdnn_aug_1a.sh]: This error was happened in my case, so I changed like this
----------
  stage14
----------
run_tdnn_aug_ernie_a.sh: creating neural net configs using the xconfig parser
tree-info exp/chain/tri5_7d_tree_aug/tree
  File "<stdin>", line 1
    print 0.5/0.1
            ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(0.5/0.1)?

**Comment by danpovey** [comment]
Oh the parens were missing.
Actually the change is OK, Kaldi already depends on python3.
